src/App.js

```diff
@@ -1,10 +1,18 @@
 import { Route, Routes } from 'react-router';
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
 import Header from './components/Header';
 import Rockets from './pages/Rocket';
 import Missions from './pages/Missions';
 import Profile from './pages/Profile';
+import { getMissions } from './redux/space/spaceSlice';
 
 function App() {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getMissions());
+  }, []);
```

Comment:
:radio_button:  Based on the requirements, you should only fetch the missions after the user navigates to the missions page. The missions are automatically fetched by placing the getMissions inside the useEffect. Find a way to implement that.

Please confirm the Project requirements, but here is an excerpt from it.

- NOTE: Make sure you only dispatch those actions once and do not add data to store on every re-render (i.e. when changing views / using navigation).

- NOTE: Rockets is the default view, so you must fetch rockets data when the application starts. However, the missions data should only be fetched (once) when a user navigates to the Missions section.